lang/en_US.json

```diff
@@ -458,5 +458,8 @@
     "Hash Comments": "Hash Comments",
     "Surprise me": "Surprise me",
     "Claim Comments - {{gameTitle}}": "Claim Comments - {{gameTitle}}",
-    "Claim Comments": "Claim Comments"
+    "Claim Comments": "Claim Comments",
+    "Rank": "Rank",
+    "User": "User",
+    "Top Achievers - {{gameTitle}}": "Top Achievers - {{gameTitle}}"
```

Comment:
Have we given up trying to keep this alphabetized?

Reply:
I think it's ok for us to do it, but if we do, I'd like to see it done automatically by ESLint so it doesn't become one more thing to remember.

I know of at least one ESLint rule that can probably do this for us. Let's push to a subsequent PR as this will likely contain a lot of noise.